src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlKeys.cs

```diff
@@ -23,15 +23,15 @@ public List<XmlPluginKey> PluginKeys
         { get; set; }
 
         [XmlElement(ElementName = "DynamicKey")]
-        public List<XmlDynamicKey> DynamicKeys
+        public List<XmlMultiStepKey> MultiStepKeys
```

Comment:
In order to free up the name "DynamicKey" for my new code, I renamed the old version to "MultiStepKey". It would be ideal to deprecate "MultiStepKey" when the new more advanced "DynamicKey" is released.